packages/c/circliful.json

```
@@ -7,14 +7,13 @@
   },
   "license": "MIT",
   "autoupdate": {
-    "source": "git",
-    "target": "git://github.com/pguso/jquery-plugin-circliful.git",
+    "source": "npm",
+    "target": "js-plugin-circliful",
```

```suggestion
    "source": "git",
    "target": "git://github.com/pguso/jquery-plugin-circliful.git",
```
The [git repo actually has more recent versions](https://github.com/pguso/js-plugin-circliful/tree/2.0.17/dist) than the [NPM package](https://www.npmjs.com/package/js-plugin-circliful), so let's stick to the git repo for the source-of-truth.